source/core/module_typeHelpers.js

```diff
@@ -278,7 +284,15 @@
             return Module._TypeRef_IsAnalogWaveform(typeRef) !== 0;
         };
 
+        Module.typeHelpers.isJSObjectRefnum = function (typeRef) {
+            return Module._TypeRef_IsJavascriptRefnum(typeRef) !== 0;
+        };
+
         var typeHandlers = [
+            {
```

Comment:
some items in this list have significance in ordering, is there any significance in ordering for isJSObjectRefnum? If so can we document that

Reply:
@gleonoliva Maybe we should document each entry? Just from looking it is not obvious where ordering is significant.

Reply:
I'm not sure if it necessary to add description to every single entry, it would be nice though.

Reply:
Will document. Basically, JavascriptRefnum should be first because it should not be treated as Intger.